CMakeLists.txt

```diff
@@ -721,6 +721,22 @@ if (WIN32 AND CLANG AND OPENMP_BUILD)
     target_link_libraries (libtesseract PRIVATE ${OpenMP_LIBRARY})
 endif()
 
+# fix for that motioned in https://github.com/tesseract-ocr/tesseract/issues/3680
+#
+# 2021-12-18 14:56:11 UTC  leleliu008@gmail.com
+#
+# Reference:
+# https://github.com/Kitware/CMake/blob/master/Modules/Platform/Android-Determine.cmake
+# https://android.googlesource.com/platform/ndk/+/master/build/cmake/android.toolchain.cmake
+# https://developer.android.com/ndk/guides/cpu-features#the_google_cpu_features_library
+# https://github.com/google/cpu_features#android-ndks-drop-in-replacement
```

Comment:
```suggestion
```

Reply:
Usually we don't add such details to the code, only short comments where needed. All that information is also in the commit message, so it does not get lost.